ResourceStatusSystem/DB/ResourceManagementDB.py

```diff
@@ -421,30 +435,39 @@ def select( self, table, params ):
 
     :return: S_OK() || S_ERROR()
     '''
-    #FIXME: this stuff about META and columns ... probably for the web?
 
     session = self.sessionMaker_o()
     table_c = getattr(__import__(__name__, globals(), locals(), [table]), table)
 
+    columnNames = []
+
     try:
       select = session.query(table_c)
       for columnName, columnValue in params.iteritems():
-        if not columnValue:
-          continue
-        column_a = getattr(table_c, columnName.lower())
-        if isinstance(columnValue, (list, tuple)):
-          select = select.filter(column_a.in_(list(columnValue)))
-        elif isinstance(columnValue, basestring):
-          select = select.filter(column_a == columnValue)
-        elif isinstance(columnValue, datetime.datetime): #FIXME: iis it correct/enough? (should check also below)
-          select = select.filter(column_a == columnValue)
-        else:
-          self.log.error("type(columnValue) == %s" %type(columnValue))
+        if columnName.lower() == 'meta' and columnValue: # special case
```

Comment:
Is this required? from line 465 to 468 will do the work .
In that case columnValue['columns'] will be objects no?
```python
so you need columnNames = [str(column) for column in columnValue['columns'] ]
```

Reply:
What is put in `columnNames` in 
```
finalResult['Columns'] = columnNames
```
is a list of strings.

Reply:
ok